tests/dist/chats.spec.dev.js

```diff
@@ -229,58 +237,59 @@ describe('chat resolvers', function () {
           case 0:
             nameForNewUser = shortid.generate();
             email = ''.concat(nameForNewUser, '@test.com');
             _context5.next = 4;
             return regeneratorRuntime.awrap(
-            axios.post(URL, {
-              query: '\n              mutation {\n                  signUp(data: {\n                  firstName:"'
-                .concat(nameForNewUser, '",\n                  lastName:"')
-                .concat(nameForNewUser, '"\n                  email: "')
-                .concat(
-                  email,
-                  '"\n                  password:"password"\n                  }) {\n                  user{\n                    _id\n                  }\n                  accessToken\n                  }\n              }\n              '
-                ),
-            })
-          );
+              axios.post(URL, {
+                query: '\n              mutation {\n                  signUp(data: {\n                  firstName:"'
+                  .concat(nameForNewUser, '",\n                  lastName:"')
+                  .concat(nameForNewUser, '"\n                  email: "')
+                  .concat(
+                    email,
+                    '"\n                  password:"password"\n                  }) {\n                  user{\n                    _id\n                  }\n                  accessToken\n                  }\n              }\n              '
+                  ),
+              })
+            );
 
           case 4:
             createNewUserResponse = _context5.sent;
             signUpData = createNewUserResponse.data;
             newUserId = signUpData.data.signUp.user._id;
             _context5.next = 9;
             return regeneratorRuntime.awrap(
-            axios.post(
-              URL,
-              {
-                query: '\n      mutation{\n          createGroupChat(data: {\n            title: "This is a group chat for testing"\n            organizationId: "'
-                  .concat(createdOrgId, '"\n            userIds: ["')
-                  .concat(loggedInUserId, '", "')
-                  .concat(
-                    newUserId,
-                    '"]\n          }){\n            _id\n          }\n        }\n              '
-                  ),
-              },
-              {
-                headers: {
-                  Authorization: 'Bearer '.concat(token),
+              axios.post(
+                URL,
+                {
+                  query: '\n      mutation{\n          createGroupChat(data: {\n            title: "This is a group chat for testing"\n            organizationId: "'
+                    .concat(createdOrgId, '"\n            userIds: ["')
+                    .concat(loggedInUserId, '", "')
+                    .concat(
+                      newUserId,
+                      '"]\n          }){\n            _id\n          }\n        }\n              '
+                    ),
```

Comment:
It would be ideal to use a template literal here to improve the readability of this query.